src/App.js

```diff
@@ -97,6 +98,16 @@ function App() {
   const [airQualityValue, setAirQualityValue] = useState(null);
   const [airQualityDesc, setAirQualityDesc] = useState("");
   const [barColor, setBarColor] = useState("transparent");
+  const [data, setData] = useState(null);
+
+  useEffect(()=> {
+    fetch("http://api.openweathermap.org/data/2.5/forecast/daily?q="+city+"&units=metric&cnt=7&appid=" + process.env.REACT_APP_APIKEY)
+      .then((res)=>{ console.log(res); return res.json(); })
```

Comment:
try replacing http with https to make it work on netifly deployment

Reply:
> try replacing http with https to make it work on netifly deployment

@himanshu-thakur-7 Done